components/alert/alert-toast.js

```diff
@@ -357,21 +383,20 @@ class AlertToast extends LitElement {
 				this._state = states.CLOSING;
 			}
 			requestAnimationFrame(() => {
-				const height = this._innerContainer.offsetHeight;
 				const bottom = parseFloat(getComputedStyle(this._innerContainer).getPropertyValue('bottom'));
 
 				if (reduceMotion || this._state === states.PREOPENING) {
 					this._state = states.CLOSED;
 					this._bottomHeight = 0;
-					this._bottomMargin = 0;
+					this._numAlertsBelow = 0;
```

Comment:
This was an easier way to compute what the bottom spacing between should work out to when there's the different sizes at play